java/src/plume/LimitedSizeSet.java

```diff
@@ -45,17 +49,23 @@
    *
    * @param max_values the maximum number of values this set will be able to hold; must be positive
    */
-  public LimitedSizeSet(int max_values) {
+  public LimitedSizeSet(/*@Positive*/ int max_values) {
     if (assertsEnabled && !(max_values > 0)) {
       throw new IllegalArgumentException("max_values should be positive, is " + max_values);
     }
     // this.max_values = max_values;
-    @SuppressWarnings("unchecked")
-    /*@Nullable*/ T[] new_values_array = (/*@Nullable*/ T[]) new /*@Nullable*/ Object[max_values];
+    @SuppressWarnings({
+      "unchecked",
+      "index", // https://github.com/kelloggm/checker-framework/issues/174
+      "value"
```

Comment:
This is a new `@SuppressWarnings` (for "value"), but there is no comment about it.